### DISCUSSION TITLE
Props - Capitolo 3

### DISCUSSION THREAD

**Discussion by GiuseppeS83** [lifeisfoo/react-guida-pratica#19] [Q&A]
Ciao a tutti,
nel capitolo 3 le "props" vanno inserite nel file App.jsx presente nella cartella ch02?

**Comment by lifeisfoo** [comment]
Ciao @GiuseppeS83,
non so a quale pagina ti stai riferendo però sì, se segui il libro attivamente modificando anche il codice allora devi partire da quello del capitolo prima. Nel caso delle attività descritte nel capitolo 3 si parte dal codice presente in ch02.
Naturalmente puoi fare copia e incolla di quella cartella e chiamarla cap3-wip o cose simili così per distinguerla da ch03 che contine la versione finale del codice presente nel capitolo 3.

Questo che ti sto scrivendo è comunque già presente nel paragrafo "Il codice" dell'introduzione.
Spero di esserti stato d'aiuto.
